examples/benchmarks/bubblebench/baseline/tsconfig.json

```diff
@@ -1,11 +1,9 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
-	"exclude": ["dist", "node_modules"],
+	"extends": "../../../../common/build/build-common/tsconfig.node16.json",
+	"exclude": ["lib", "node_modules"],
```

Comment:
Please carefully review the differences from this change.
(I have avoided doing so because I try to stick to one change at a time.)

It would be good to capture the compilerOptions changes in PR description. Thanks!

Reply:
I saw the description update. Thanks.
But there is one more difference.
With this change we have
  "lib": ["ES2020", "DOM", "DOM.Iterable"],
which adds "DOM.Iterable" to the list.
I don't think that is an issue but good to note.